specification/communication/data-plane/MediaComposition/preview/2022-07-16-preview1/mediacomposition.json

```diff
@@ -0,0 +1,1180 @@
+    "version": "2022-07-16-preview1"
+  },
+  "paths": {
+    "/mediaCompositions/{mediaCompositionId}": {
+      "get": {
+        "tags": [
+          "MediaComposition"
+        ],
+        "summary": "Gets a media composition",
+        "operationId": "MediaComposition_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "mediaCompositionId",
+            "description": "",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/MediaCompositionBody"
+            }
+          },
+          "400": {
+            "description": "Bad Request.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "401": {
+            "description": "Unauthorized.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "403": {
+            "description": "Forbidden.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "429": {
+            "description": "Too many requests.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "503": {
+            "description": "Service unavailable.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Get Media Composition": {
+            "$ref": "./examples/MediaComposition_GetMediaComposition.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "MediaComposition"
+        ],
+        "summary": "Creates a new media composition",
+        "operationId": "MediaComposition_Create",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "mediaCompositionId",
+            "description": "",
+            "required": true,
+            "type": "string"
```

Comment:
Consider specifying a maximum length (maxLength) and characters allowed (pattern) for this path parameter.